### PR TITLE
zh-translation: faq/distributed-tracing/why-mixer-spans.md

### PR DESCRIPTION
**Pull request by helight**
zh-translation: faq/distributed-tracing/why-mixer-spans.md
ref:https://github.com/servicemesher/istio-official-translation/issues/1398

### CODE REVIEW

File: content/zh/faq/distributed-tracing/why-mixer-spans.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Why do I see `istio-mixer` spans in some of my distributed traces?
+title: 为什么在我的一些分布式追踪中会有 `istio-mixer` span？
 weight: 100
 ---
 
-Mixer generates application-level traces for requests that reach Mixer with tracing headers. Mixer generates spans, labeled `istio-mixer` for any critical work that it does, including dispatch to individual adapters.
+Mixer为到达 Mixer 并且带有追踪头的请求生成了应用级别的追踪。Mixer 为它做的任何关键工作都生成 span 并且打上了 `istio-mixer` 标签，包括分发到各个适配器。
```
Comment (gorda) [inline content/zh/faq/distributed-tracing/why-mixer-spans.md]: > Mixer为

建议：Mixer 为

Comment (helight) [reply content/zh/faq/distributed-tracing/why-mixer-spans.md]: accept

**Pull request merged**